framework/include/media/FocusManager.h

```diff
@@ -0,0 +1,102 @@
+/* ****************************************************************
```

Comment:
This is opened to user, so description is very important.
If you decide get id from application, you should mention it in description.

Reply:
I removed `setId` and make id automatically